views/pwas/view.hbs

```diff
@@ -48,6 +48,8 @@
         <div class="pwabox-body">
           <h3>{{pwa.name}}</h4>
           <p>{{pwa.description}}</p>
+          <p>Added: {{pwa.created}}</p>
```

Comment:
moment.js [does relative dates](http://momentjs.com/docs/#/displaying/fromnow/), if you want to escalate to that…